src/profile-logic/function-info.js

```diff
@@ -28,21 +30,21 @@ function findAddonForChromeURIHost(host) {
   });
 }
 
-function ensureResource(name, resourceDescription) {
+function ensureResource(name: string, resourceDescription): string {
```

Comment:
`resourceDescription` miss a description. We should create a new better type, that ideally would be an union of all the types (one for 'library', one for 'addon', etc).

But I think all these functions are actually not used at all. I think @mstange imported this file from our previous perf project but finally never used it :)

So I think it's better to not invest too much time in these functions. It's good enough that it passes. Maybe we should just delete all these unused functions, what do you think @mstange ?

Reply:
Yeah, I couldn't find any usage of this, which is why I only typed the easy cases :) So I'll leave it like this for now, and we'll see later what @mstange says.

Reply:
If we aren't using them, I would suggest we delete them and not bother with them. The old Cleopatra codebase is still around if we need to reference them.